src/com/lftechnology/java/training/alina/basic/Student.java

```diff
@@ -0,0 +1,116 @@
+package com.lftechnology.java.training.alina.basic;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Student Class consists of all the details of student's & their performance
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class Student extends Course implements Comparable<Course> {
+
+	static ArrayList<Student> students = new ArrayList<Student>();
+	static ArrayList<Course> courses = new ArrayList<Course>();
+
+	private static final Logger LOGGER = Logger.getLogger(Student.class.getName());
+	private String fullName;
+	private int standard;
+	private int rollNo;
+
+	public Student(float subject1, float subject2, float subject3,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)